### DISCUSSION TITLE
23.12.28 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#97] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - nGrinder 설치
    - 스크립트 작성 및 성능 테스트
- 내일은 어떤 일을 하나요?
    - 발표 준비

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - nGrinder 학습
    - docker 사용
  - 스크립트 오류 해결
  - 부하 테스트 진행
- 내일은 어떤 일을 하나요?
  - 발표 준비
